lib/evss/configuration.rb

```diff
@@ -42,7 +42,7 @@ def connection
         faraday.use      Faraday::Response::RaiseError
         faraday.response :betamocks if mock_enabled?
         faraday.response :snakecase, symbolize: false
-        faraday.response :json
+        faraday.response :json, content_type: /\bjson$/
```

Comment:
So if the response `content-type` is not `application/json`, it will not try to parse the response?  

A comment or wrapping `content_type: content_type: /\bjson$/` this in a developer-readable method that explains more as to what is going on would help.  E.g
```ruby
def do_not_parse_non_json_or_something_i_guess
  content_type: /\bjson$/
end
```

Reply:
yes, it won't try to parse non-json content and will pass the response unmodified on to the next middleware in the chain. 👍 I'll add a comment to explain the setting.